#!/bin/sh /usr/share/dpatch/dpatch-run
## 327-magic-add-svg.dpatch by Noel Torres <tecnico@ejerciciosresueltos.com>
##
## DP: Adds magic for SVG (Closes: #417331).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/sgml file-4.20/magic/Magdir/sgml
--- file-4.20.orig/magic/Magdir/sgml	2007-01-19 21:09:07.000000000 +0000
+++ file-4.20/magic/Magdir/sgml	2007-05-17 16:54:52.000000000 +0000
@@ -33,3 +33,9 @@
 0	string	#\ HTTP\ Cookie\ File	Web browser cookie text
 0	string	#\ Netscape\ HTTP\ Cookie\ File	Netscape cookie text
 0	string	#\ KDE\ Cookie\ File	Konqueror cookie text
+
+# Type:	SVG Vectorial Graphics
+# From:	Noel Torres <tecnico@ejerciciosresueltos.com>
+0	string	\<?xml\ version="
+>15	string	>\0
+>>23	regex	.*\<svg			SVG Scalable Vector Graphics image
